### PR TITLE
Created a Shrug Command

### PR DESCRIPTION
**Pull request by Vault108**
Added a shrug command

### CODE REVIEW

File: sopel/modules/shrug.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+import sopel.module
+@sopel.module.commands('shug')
```
Comment (dgw) [inline sopel/modules/shrug.py]: Typo…

Comment (Vault108) [inline sopel/modules/shrug.py]: Fixed.

File: sopel/modules/shrug.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+import sopel.module
```
Comment (dgw) [inline sopel/modules/shrug.py]: Best to `from sopel.module import commands` here and then just use `@commands()` below.

File: sopel/modules/shrug.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+
+@sopel.module.commands('shrug')
```
Comment (dgw) [inline sopel/modules/shrug.py]: If you tweak the import above as suggested, this can just be `@commands('shrug')`.

**Comment by alanhuang122** [comment]
FWIW, I've found a number of little macros of this sort (e.g. "ಠ_ಠ") to be useful. I propose putting this and any future additions in an `emoticons.py` module, instead of having individual files for individual emoticons.

**Comment by dgw** [comment]
@alanhuang122 Great point! But let's first see if @Vault108 is still interested in working on this and decide the course of action from there.

**Comment by dgw** [comment]
Closing in favor of #1332.

**Pull request closed without merging**